content/blog/how-we-work

```diff
@@ -0,0 +1,31 @@
++++
+title = "How we work"
+heading = "Understanding the Organization for Ethical Source’s structure and governance model"
+summary = "This is the first post of a series explaining the Organization for Ethical Source’s structure, governance model, practices, and norms."
++++
+
+<time>May 3, 2021</time>
+
+As you know, the Ethical Source movement [organized into a legal entity](/blog/oes-announcement/) in late January.
```

Comment:
I would take out the "As you know", and maybe move this sentence to the first paragraph